tests/mock-documents/delegated-ocap-root-alpha.js

```diff
@@ -1,5 +1,8 @@
 module.exports = {
-  "@context": "https://w3id.org/security/v2",
+  "@context": [
+    "https://w3id.org/zcap/v1",
+    "https://w3id.org/security/suites/ed25519-2018/v1"
```

Comment:
Would prefer we update to 2020 here, but understand if that's done in a separate PR.

Reply:
Yup, separate PR.